docs/namespaces/lyrics.rst

```diff
@@ -29,3 +29,29 @@ The required ``value`` field can contain arbitrary text data, e.g., lyrics.
 .. note::
     ``confidence`` is an unconstrained field, and may contain arbitrary data.
 
+lyrics_bow
+~~~~~~~~~~
+Time-aligned bag-of-words or bag-of-ngrams.
+
+    ===== ======== ====== ==========
+    time  duration value  confidence
+    ===== ======== ====== ==========
```

Comment:
Sorry for the spam. Maybe this project should move to reviewable too?
Thought that it was way more intuitive :)